grape-entity.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = 'grape-entity'
 
-  s.add_runtime_dependency 'activesupport'
+  s.add_runtime_dependency 'activesupport', '<= 4.2.6'
```

Comment:
I think `< 5` is the right entry here. I am however worried that we're effectively saying we're not compatible with activesupport 5, while the code is and the problems are structural. Maybe we should just fix things around here to work with AS 5, like increment the minimum Ruby version?